Makefile

```diff
@@ -22,13 +44,13 @@ bundle-openapi:
 	redocly bundle openapi/versions/v1.json -o docs/openapi/v1.json
 
 
-# Deployment
+# Terraform base command
```

Comment:
please add bit more context on why we are doing this.

Reply:
updated in b6b119a6dd73086ee41929ddbaedf2483cca9b9c